developer-docs-site/docs/index.md

```diff
@@ -8,7 +8,9 @@ hide_table_of_contents: true
 
 # Aptos Developer Documentation
 
-Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
+Welcome! At Aptos Labs, we are building a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." We at Aptos Labs are here to help you grow a [web3 ecosystem](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family. 
```

Comment:
This is technically owned by the Foundation. We just support it as the Labs. So we should make sure this is reflected as such. Let's make an issue here to follow up and ensure we respect the Foundation appropriately.

Reply:
> respect the Foundation

Issue filed and assigned to us:
https://github.com/aptos-labs/aptos-core/issues/5773